### DISCUSSION TITLE
Struts 1.6.0

### DISCUSSION THREAD

**Discussion by MarcSchiering** [ste-gr/struts1-reloaded#29] [General]
Hello Stefan Graff,

we have been following the excellent work you and your team have been doing in developing Struts for quite some time now. Your efforts have consistently impressed us.

As a small software company, we currently use Struts 1.3.10 in our projects. With the transition to Tomcat 10 looming ahead, we are faced with the challenge of upgrading our applications. It is crucial for us to know when you plan to release Struts 1.6.0.

Could you please inform us if there is already a schedule in place for the release of Struts 1.6.0 or if you have any information regarding when we can expect the availability of this version?

We would greatly appreciate your support and insight.

Thank you in advance for your time and effort.

Best regards, 
Marc Schiering

**Comment by ste-gr** [comment]
Hallo Marc,

thank you very much for your praise. I am delighted that you like my work.

You can already use version 1.5.0-RC2 under Tomcat 10.0. I think this version should also work under Tomcat 10.1.

The only open point for version 1.5.0 is the missing TLD documentation ([taglib-maven-plugin](https://github.com/weblegacy/taglib-maven-plugin)), which I am currently adapting for Jakarte 9+. But that will take some more time.

Greetings
Stefan

**Reply by MarcSchiering** [reply]
Hi, unfortunately version 1.5.0-RC2 does not work under Tomcat 10.1.

Tomcat 10.1 requires at least Java 11 and the Servlet Spec is 6.0, JSP Spec 3.1, ....

See https://tomcat.apache.org/whichversion.html

**Reply by ste-gr** [reply]
Hi @MarcSchiering,

did you get an error-message?
I have just now run the IT tests under Tomcat 10.1.19 with JDK 17 and there were no problems.

Greetings
Stefan
